### ISSUE TITLE
Update Web API scaffolder overposting link

### ISSUE THREAD

**Issue by Rick-Anderson** [aspnet/Scaffolding#1265]
When you scaffold a model for Web API, the following comment is generated

```
        // PUT: api/TodoItems/5
        // To protect from overposting attacks, please enable the specific properties you want to bind to, for
        // more details see https://aka.ms/RazorPagesCRUD.
```

https://aka.ms/RazorPagesCRUD suggests using [TryUpdateModelAsync](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.tryupdatemodelasync?view=aspnetcore-3.1#Microsoft_AspNetCore_Mvc_ControllerBase_TryUpdateModelAsync_System_Object_System_Type_System_String_), which is only available when posting from a form. `TryUpdateModelAsync` is typically the wrong approach for REST APIs.  See [this GitHub comment](https://github.com/aspnet/Mvc/issues/5659#issuecomment-270717629) by `@Eilon`

Web API scaffolding should point to https://docs.microsoft.com/aspnet/core/tutorials/first-web-api#over-post

Also, per MS style guide, remove `please`. Update RP (Razor Pages) comment and remove `please`.

Any PR that produces user output (exception messages, console output, etc), please mention @Rick-Anderson

**Comment by deepchoudhery** [comment]
Will update. I'll use an fwlink so I don't have to update the templates. Will be fixed for 5.0-preview2 release.

**Comment by Rick-Anderson** [comment]
@deepchoudhery  sounds great. Note I corrected the link and removed the `/en-en/` language.
